submissions/LuckyDnepr/Hooli-popup-task/index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles-menu.css">
+    <title>Hooli menu</title>
+</head>
+
+<body>
+    <header class="header">
+        <input id="nav_keeper_input" type="checkbox" tabindex="1">
+        <div class="nav_keeper_lines">
```

Comment:
You can use `<label>` tag instead of hide hide checkbox under div 

The example - [Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label)
How to hide checkbox correct - [YouTube](https://www.youtube.com/watch?v=E6kLaaQFctU&ab_channel=VadimMakeev)